rust_src/src/fns.rs

```diff
@@ -306,4 +308,39 @@ pub extern "C" fn internal_equal_misc(
     }
 }
 
+/// Return list of 1 minute, 5 minute and 15 minute load averages.
+///
+/// Each of the three load averages is multiplied by 100, then converted
+/// to integer.
+///
+/// When USE-FLOATS is non-nil, floats will be used instead of integers.
+/// These floats are not multiplied by 100.
+///
+/// If the 5-minute or 15-minute load averages are not available, return a
+/// shortened list, containing only those averages which are available.
+///
+/// An error is thrown if the load average can't be obtained.  In some
+/// cases making it work would require Emacs being installed setuid or
+/// setgid so that it can read kernel information, and that usually isn't
+/// advisable.
+#[lisp_fn(min = "0")]
+pub fn load_average(use_floats: bool) -> Vec<LispObject> {
+    let mut load_avg: [libc::c_double; 3] = [0.0, 0.0, 0.0];
+    let loads = unsafe { libc::getloadavg(load_avg.as_mut_ptr(), 3) };
```

Comment:
This is a Unix call. It works on Mac, Linux, BSD, etc. But it will bomb on Windows.

Reply:
There is code in w32.c to emulate it. You could wrap this in a `cfg!()` check.